rllib/core/models/configs.py

```diff
@@ -107,6 +108,51 @@ def build(self, framework: str = "torch") -> Model:
             return TfMLPHead(self)
 
 
+@ExperimentalAPI
+@dataclass
+class CNNEncoderConfig(ModelConfig):
+    """Configuration for a convolutional network.
+
+    See ModelConfig for usage details.
```

Comment:
I think it's worth giving a concrete CNN example and explaining the stack (e.g. DQN paper architecture)
For example How would you describe this chain (where input images are represented as a 4x84x84 volume of 4 stacked grayscale images)
```
Conv2d(in_channels=4, out_channels=32, kernel_size=8, stride=4)
ReLU()
Conv2d(in_channels=32, out_channels=64, kernel_size=4, stride=2)
ReLU()
Conv2d(in_channels=64, out_channels=64, kernel_size=3, stride=1)
ReLU()
Flatten(),
Linear(in_features=3136, out_features=512)
```